incubator/binding-grpc-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/kafka/streams/kafka/server.stream.rpc.read.aborted/server.rpt

```diff
@@ -40,18 +40,12 @@ read zilla:data.ext ${kafka:matchDataEx()
                                .header("zilla:identity", "test")
                                .header("zilla:service", "example.EchoService")
                                .header("zilla:method", "EchoServerStream")
-                               .header("zilla:request", "UNARY")
-                               .header("zilla:response", "STREAM")
                                .header("zilla:correlation-id", "59410e57-3e0f-4b61-9328-f645a7968ac8-d41d8cd98f00b204e9800998ecf8427e")
                                .build()
                            .build()}
-read ${grpc:protobuf()
-           .string(1, "Hello World")
-           .build()}
+read zilla:data.null
 
-
-read closed
-write close
+read aborted
```

Comment:
Yes, if we receive an abort from grpc we want to propagate the end of the request otherwise we may have request in never-ending state  on kafka-grpc side. Lets say if we haven't send anything yet it and sending end of request it will be ignored on the kafka-grpc side.